sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -617,6 +619,80 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates blocked disbursement notification for student.
+   * @param notification notification details.
+   * @param userId student's user id associated with this notification.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  async saveDisbursementBlockedNotificationForStudent(
+    notification: DisbursementBlockedNotificationForStudent,
```

Comment:
For student notification, we can re-use an existing interface `StudentNotification` which has userId in it so that we don't have to pass userId as param seperately.

Reply:
Aren't we trying to keep a separate interface per notification?